components/Blueprints/class-runner.php

```diff
@@ -383,7 +383,7 @@ private function load_blueprint() {
 		if ( function_exists( 'mb_check_encoding' ) ) {
 			$is_valid_utf8 = mb_check_encoding( $blueprint_string, 'UTF-8' );
 		} else {
-			$is_valid_utf8 = utf8_is_valid_byte_stream( $blueprint_string );
+			$is_valid_utf8 = ! _wp_has_noncharacters_fallback( $blueprint_string );
```

Comment:
`wp_has_noncharacters()` from `utf8.php` is indeed much faster. unless there is a reason you need them, I would suggest sticking to the non-private methods if you can.